src/odemis/driver/hamamatsurx.py

```diff
@@ -0,0 +1,2043 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 14 Jun 2016
```

Comment:
That's probably not the right date ;-)

Reply:
That was already fixed but got lost when reorganizing my commits :/ Will be now!